cvat/apps/engine/tests/test_rest_api_3D.py

```diff
@@ -208,6 +211,24 @@ def etree_to_dict(t):
 
 
 class Task3DTest(_DbTestBase):
+    def compare_annotations(self, annos_orig: list[dict], annos_imported: list[dict]):
+
+        for _type in set(AnnotationType):
+            key = f"{_type}s"
+            anns = annos_imported.data[key]
+            self.assertEqual(
+                len(annos_orig.data[key]),
+                len(annos_imported.data[key]),
+                "Different number of shapes"
+            )
+            for ann in anns:
+                self.assertEquals(ann.get('source'), 'file')
+        compare_objects(self,
+                        annos_orig.data, annos_imported.data,
+                        ignore_keys=["source", "id", "version"],
```

Comment:
There are many instances of id field cleanups in the tests in this file. They can probably be removed now.

```
task_ann_prev.data["shapes"][0].pop("id")
task_ann.data["shapes"][0].pop("id")
```